### ISSUE TITLE
What does ignore_lb = 255 do?

### ISSUE THREAD

**Issue by wusaifei** [CoinCheung/pytorch-loss#2]
What does ignore_lb = 255 do? Please help me to explain. Thank you very much for your help.

**Comment by CoinCheung** [comment]
It means the samples with label=255 with be ignored when computing the gradients. You can refer to pytorch official F.nll_loss for similar usage.
